.eslintrc

```diff
@@ -0,0 +1,186 @@
+{
+    "ecmaFeatures": {},
+    "parser": "espree",
```

Comment:
You can remove this as you always want to use the eslint default parser. And as of now its espree. But I think we dont have to mention this.